src/main/java/org/opentestsystem/ap/ims/service/export/writer/ImageUtil.java

```diff
@@ -15,8 +15,13 @@ public static File createImageFromSVG(File directory, File file, float width, fl
         PNGTranscoder t = new PNGTranscoder();
 
         // Set the transcoding hints.
-        t.addTranscodingHint(PNGTranscoder.KEY_WIDTH, width);
-        t.addTranscodingHint(PNGTranscoder.KEY_HEIGHT, height);
+        if (width > 0) {
```

Comment:
I left the logic that if a height and width is passed in the image it will be respected.